docs/experiments-monot5-gpu.md

```diff
@@ -11,19 +11,30 @@ We rerank the BM25 run files that contain ~1000 passages per query using monoT5.
 monoT5 is a pointwise reranker. This means that each document is scored independently using T5.
 
 ## Environment Setup
+This experiment is proved to work on Tensorflow-2.3.0. Please make sure you have required gcc version accourding to [TensorFlow gpu dependency](https://www.tensorflow.org/install/source#gpu). 
+
+
+```
+module list             # Check if you have coreect version of gcc
```

Comment:
is this required for a particular environment/cluster? If so, I would not add it here as not everybody will have the same problem

Reply:
hmm, true. I am not sure if this will happened for everyone. Let me add a trouble shooting section and move it there

Reply:
added a trouble shooting section to include those issue